components/onboarding_explore_tools_tour/explore_tools_tour_manager.tsx

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {getInt} from 'mattermost-redux/selectors/entities/preferences';
+import {savePreferences, savePreferences as storeSavePreferences} from 'mattermost-redux/actions/preferences';
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/common';
+
+import {setProductMenuSwitcherOpen} from 'actions/views/product_menu';
+import {trackEvent as trackEventAction} from 'actions/telemetry_actions';
+
+import {GlobalState} from 'types/store';
+
+import {
+    generateTelemetryTag,
+    OnboardingTaskCategory,
+    OnboardingTaskList,
+    OnboardingTasksName,
+} from 'components/onboarding_tasks';
+
+import {
+    AutoTourStatus,
+    OtherToolsTour,
+    ExploreOtherToolsTourSteps,
+    FINISHED,
+    SKIPPED,
+    TTNameMapToATStatusKey,
+    TutorialTourName,
+} from '../onboarding_tour/constant';
+import {isKeyPressed, KeyCodes} from '../onboarding_tour/utils';
+
+export type ActionType = 'next' | 'prev' | 'dismiss' | 'jump' | 'skipped'
+
+export interface OnBoardingTourTipManager {
```

Comment:
Discussed offline that we're ok leaving this as a separate implementation from `onboarding_tour_manager.tsx`'s `useOnBoardingTourTipManager`, given the possibility of the tours diverging further. As we are leaving it, could we rename some of the things in here?
* `OnBoardingTourTipManager` -> `ExploreToolsTourTipManager`
* `useOnBoardingTourTipManager` -> `useExploreToolsTourTipManager`